server/vulnerabilities/msrc/io/security_bulletin_name.go

```diff
@@ -0,0 +1,57 @@
+package io
+
+import (
+	"errors"
+	"strings"
+	"time"
+)
+
+const (
+	MSRCFilePrefix = "fleet_msrc_"
```

Comment:
this name is not used outside `io`, but it's exported

Reply:
Not now but it will in the next PR.